Geometry/CaloEventSetup/python/CaloGeometryBuilder_cfi.py

```diff
@@ -10,4 +10,12 @@
                                 'TOWER'           )
 )
 
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
+run3_common.toModify(CaloGeometryBuilder,
+                     SelectedCalos = ['HCAL', 'ZDC', 'EcalBarrel', 'EcalEndcap', 'EcalPreshower', 'TOWER']
+)
 
+from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+phase2_common.toModify(CaloGeometryBuilder,
+                       SelectedCalos = ['HCAL', 'ZDC', 'EcalBarrel', 'TOWER', 'HGCalEESensitive', 'HGCalHESiliconSensitive', 'HGCalHEScintillatorSensitive']
+)
```

Comment:
I'm not sure I understand the change related to Phase-2. Could you please clarify?

The `CaloGeometryPhaseII_cff.py` calls `CaloGeometryBuilderPhaseII_cfi` where the change is applied.  However, this introduced modifier seems to do the same. But who will call this? Or should we change the way to call, e.g. call `CaloGeometryBuilder_cfi.py` directly, so that we can remove `CaloGeometryPhaseII_cff.py` ?